lib/smack/SmackRep.cpp

```diff
@@ -55,6 +59,36 @@ bool SmackRep::isInt(const llvm::Value* v) {
   return isInt(v->getType());
 }
 
+unsigned SmackRep::getIntSize(const llvm::Value* v) {
+  assert(isInt(v) && "wrong type, should be integer.");
+  return getIntSize(v->getType());
+}
+
+unsigned SmackRep::getIntSize(const llvm::Type* t) {
+  const llvm::IntegerType* intType = llvm::cast<llvm::IntegerType>(t);
+  return intType->getBitWidth();
+}
+
+bool SmackRep::isPointer(const llvm::Type* t) {
+  return t->isPointerTy();
+}
+
+bool SmackRep::isPointer(const llvm::Value* v) {
+  return isPointer(v->getType());
+}
+
+unsigned SmackRep::getPtrSize(const llvm::Value* v) {
+  assert(isPointer(v) && "wrong type, should be pointer.");
```

Comment:
Currently this is the only call to `isPointer`. Since that is the case, there is no reason to have an `isPointer` method. Simply assert `t->isPointerTy()` in `getPtrSize(llvm::Type*)`.